app/assets/stylesheets/resource_curriculum_show.scss

```diff
@@ -9,13 +9,12 @@
   }
 }
 
-
 li {
   list-style-type: none;
   font-size: 1rem;
 }
 
-.subject_heading {
+.subject_heading, .course_heading {
```

Comment:
Selector `subject_heading` should be written in lowercase with hyphens<br>Selector `course_heading` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line